File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let answer = [];
    let a = Math.min(n, m)
    let b = Math.max(n, m)
    
    for (let i = a; i >= 1; i--) {
        if (a % i == 0 && b % i == 0) {
            return [i, (a * b / i)]
        }
    }
    return answer;
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def solution(n, m):
    a = min(n, m)
    b = max(n, m)
    
    for i in range(a, 0, -1):
        if a % i == 0 and b % i == 0:
            return [i, (a * b) // i]
    return []